mavros/src/plugins/setpoint_velocity.cpp

```diff
@@ -62,7 +69,8 @@ class SetpointVelocityPlugin : public plugin::PluginBase,
 	ros::Subscriber vel_unstamped_sub;
 	ros::ServiceServer mav_frame_srv;
 
-	uint8_t mav_frame = utils::enum_value(MAV_FRAME::LOCAL_NED);
+	std::string mav_frame_str;
```

Comment:
For what purpose store string repr in class? I think that it may be local variable.

Reply:
ok done !